libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/VoiceInstructionsObserver.kt

```diff
@@ -8,7 +8,10 @@ import com.mapbox.navigation.core.MapboxNavigation
  */
 fun interface VoiceInstructionsObserver {
     /**
-     * Called every time on a new [VoiceInstructions] is applicable
+     * Called every time on a new [VoiceInstructions] is applicable. Only new voice instructions
+     * are available via this observer.
```

Comment:
```suggestion
     * Called every time on a new [VoiceInstructions] is applicable. Only new voice instructions
     * are available via this observer because the timing of delivering and playing out an instruction is critical for correct navigation experience.
```